types/swagger-ui-dist/index.d.ts

```diff
@@ -13,13 +13,18 @@ export as namespace SwaggerUIDist;
 export function getAbsoluteFSPath(): string;
 export function absolutePath(): string;
 
+interface Url {
+  url: string;
+  name: string;
+}
+
 interface SwaggerUIBundle {
   (a: {
     deepLinking: boolean;
     dom_id: string;
     presets: any[];
     plugins: any;
-    urls: Array<[string, string]>;
+    urls: Url[];
```

Comment:
Didn't notice the document before🤔

Parameter name | Docker variable | Description
-- | -- | --
configUrl | CONFIG_URL | String. URL to fetch external configuration document from.
dom_id | DOM_ID | String, REQUIRED if domNode is not provided. The ID of a DOM element inside which SwaggerUI will put its user interface.
domNode | Unavailable | Element, REQUIRED if dom_id is not provided. The HTML DOM element inside which SwaggerUI will put its user interface. Overrides dom_id.


It seems we can do a little more on this commit:
most parameters are optional actually, (except `domNode` and `dom_id`, but we can still make it optional though),

`deepLinking?: boolean;
    dom_id?: string;	    dom_id?: string;...` should be fine

Thanks for this fix anyway, I was just about to make the same commit😁

Reply:
I didn't notice that document before either :/ 

I just added the parameters from there and made them all optional: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37115/commits/aacfe09ae0a57b5d151f075d862c3fa8104e364c

I left `domNode` and `dom_id` optional as well, because they are `REQUIRED if...` and that depends on a condition, so they're not technically "required" in the traditional sense.